pkg/controller.v1/mpi/mpijob_controller.go

```diff
@@ -125,8 +123,7 @@ type MPIJobReconciler struct {
 // Reconcile is part of the main kubernetes reconciliation loop which aims to
 // move the current state of the cluster closer to the desired state.
 func (jc *MPIJobReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	_ = log.FromContext(ctx)
-	logger := jc.Log.WithValues(kubeflowv1.MPIJobSingular, req.NamespacedName)
+	logger := jc.Log.WithValues("namespace", req.NamespacedName)
```

Comment:
If we want we can get the logger [from the context](https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.19.0#LoggerFrom) and add it with additional info from the reconciler's request with labels:

```
log := ctrl.LoggerFrom(ctx).WithValues(....)
```

Reply:
> Why did you change it ?

I think the MPIJobSingular, whose value is "mpijob", is not clear as the key of NamespacedName.
Moreover, I attached the controllerName as the logger name here (https://github.com/kubeflow/training-operator/pull/2177#discussion_r1683242378).

Are there any reasons that MPIJobSingular is used here?

Reply:
> We can get the logger [from the context](https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.19.0#LoggerFrom) and augment it with additional info from the reconciler's request with labels:

To use `LoggerFrom` method, we need to modify some methods to receive context.Context. Since it requires much more modifications, I think it is better to work on as another issue.